docs/blog/posts/release-hatch-1160.md

```diff
@@ -0,0 +1,74 @@
+---
+date: 2025-11-24
+authors: [cjames23]
+description: >-
+  Hatch v1.16.0 brings workspace support, dependency-groups, and sbom support.
+categories:
+  - Release
+---
+
+
+# Hatch v1.16.0
+
+Hatch [v1.16.0](https://github.com/pypa/hatch/releases/tag/hatch-v1.16.0) brings workspace support, dependency-groups, and sbom support.
+
+<!-- more -->
+
+## Workspace Support 
+
+Workspace support is designed to allow supporting monorepos with development by installing all sub repos as editable installs within environments and modeled after [Cargo workspaces](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html)
+
+Workspaces have members that can be included using full paths or partial paths with globs, and can exclude sub paths. 
+One design choice that users will find different from a tool like `uv` is that workspaces are configured per environment following in our philosophy that 
+hatch is environments first as configuration allowing complex configurations of environments that fit the needs of a project.
```

Comment:
1. I learned about this recently https://github.com/astral-sh/uv/blob/main/STYLE.md#styling-uv
2. Perhaps we should expand that philosophy section and/or remove mention of tox/nox.

```suggestion
Workspace members are defined by filters of relative or absolute paths, with support for glob patterns. Each member may also select which of their [features](../../config/environment/overview.md/#features) to enable.

One design choice that users will find different from a tool like uv is that workspaces are configured per environment. Adhering to Hatch's environment-first [philosophy](../../why.md/#environment-management) allows full compatibility with the environment feature set, such as matrices of workspaces with different configuration.
```